src/core.storage.js

```diff
@@ -1,44 +1,122 @@
-class Storage {
-  static create(values, defaults) {
-    const store = new Storage();
-    for (const key of Object.keys(values)) {
-      store.setIfNull(values[key], defaults[key]);
+
+class ExtStore {
+  constructor(values, defaults) {
+    this._isInit = false;
+
+    this._init = async function () {
```

Comment:
Why not just put this as class method like

```
async _init() {

}
```

Reply:
To make use of the parameters values and defaults